pom.xml

```diff
@@ -653,7 +653,7 @@
   <repositories>
     <repository>
       <id>querydsl</id>
-      <url>http://www.querydsl.com/maven</url>
+      <url>https://www.querydsl.com/maven</url>
```

Comment:
The QueryDSL repository doesn't seem to support HTTPS (see the build failure).

Perhaps we don't need this repository though? I believe all our dependencies are in Maven Central.

Reply:
```
[ERROR] Failed to execute goal on project querydsl-sql: Could not resolve dependencies for project com.querydsl:querydsl-sql:jar:5.0.0-SNAPSHOT: The following artifacts could not be resolved: com.oracle:ojdbc6:jar:11.1.0.7.0, oracle:sdoapi:jar:11.2.0, cubrid:cubrid-jdbc:jar:9.3.1.0005: Could not find artifact com.oracle:ojdbc6:jar:11.1.0.7.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```
I guess it is needed.